### PR TITLE
flpath751 - Add MTA v6.2.2 and MTA v7.0.2

### PR DESCRIPTION
**Pull request by rhkp**
Hi @dmartinol / @masayag / @pkliczewski,
Please review and merge or provide feedback for changes.

This is part 1 of instructions steps 1-4 at https://github.com/parodos-dev/serverless-workflows/tree/main?tab=readme-ov-file#to-introduce-a-new-workflow.

For part 2 with steps 5-7, I will raise a separate PR towards serverless-workflows-config repo.

Thanks.

### CODE REVIEW

File: mtav6.2.2/application.properties [hunk 1]
```diff
@@ -0,0 +1,6 @@
+quarkus.kogito.devservices.enabled=false
+
+mta.url = ${MTA_URL:http://mta-ui.openshift-mta.svc.cluster.local:8080}
+quarkus.rest-client.mta_json.url = http://mta-ui.openshift-mta.svc.cluster.local:8080
```
Comment (dmartinol) [inline mtav6.2.2/application.properties:4]: you should use `quarkus.rest-client.mta_json.url = ${mta.url}/hub` as in the original mta folder (same for the other application.properties)

File: e2e/mtav6.2.2.sh
Comment (dmartinol) [inline e2e/mtav6.2.2.sh:1]: the three files `mtaXYZ.sh` are exactly the same, can't you just use one and refer to it in the GH action?

**Review by dmartinol — COMMENTED** [review]
Shouldn't we also remove the original mta folder and the associated action?
lgtm apart from the comments I've raised (but Gabriel and Moti rules)

File: Makefile [hunk 2]
```diff
@@ -27,11 +30,7 @@ else
 	DOCKER_CONF := $(HOME)/.docker
 endif
 
-WORKDIR := $(shell mktemp -d)
-ifeq ($(shell uname),Darwin)
-	# Use a fixed folder to simplify limactl configuration (must be mounted with Write permissions)
-	WORKDIR := /tmp/serverless-workflows
-endif
+WORKDIR := ~/workdir
```
Comment (dmartinol) [inline Makefile:33]: are you sure this works when it's launched from a GH action, have you tested this use case? (e.g., "escalation" workflows are built this way)

File: Makefile [hunk 3]
```diff
@@ -166,13 +165,4 @@ gen-manifests: prepare-workdir
 	cd $(WORKDIR)
 	@$(CONTAINER_ENGINE) run --rm -v $(WORKDIR):/workdir -w /workdir \
 		$(LINUX_IMAGE) /bin/bash -c "${SCRIPTS_DIR}/gen_manifests.sh $(WORKFLOW_ID) $(ENABLE_PERSISTENCE)"
-	@echo "Manifests are available in workdir $(WORKDIR)/$(WORKFLOW_ID)/manifests"
-
-# Target: push-manifests
```
Comment (dmartinol) [inline Makefile:171]: any reasons for removing this target?
as I said before, "escalation" is built with the Makefile and manifests pushed with this target

File: mtav7.0.2/application.properties [hunk 4]
```diff
@@ -0,0 +1,6 @@
+quarkus.kogito.devservices.enabled=false
+
+mta.url = ${MTA_URL:http://mta-ui.openshift-mta.svc.cluster.local:8080}
+quarkus.rest-client.mta_json.url = ${mta.url}
```
Comment (dmartinol) [inline mtav7.0.2/application.properties:4]: if you look at the original file in mta folder, it is:
`${mta.url}/hub`

File: e2e/mtav6.2.2.sh
Comment (dmartinol) [reply e2e/mtav6.2.2.sh:1]: @rhkp no replies?

**Comment by rhkp** [comment]
Hi @dmartinol Thanks for review comments. I am working on making local testing of workflows ( please see directory experimental-ci-on-vm) on local clusters work first. I think I will switch this PR to draft or close for now and raise a new one.

File: mtav7.0.2/application.properties (hunk 4 above)
Comment (rhkp) [reply mtav7.0.2/application.properties:4]: Please check: https://docs.google.com/document/d/1Zmv8mGSKA_fQzC3J38d5jYaJ28EmGTIRE8jIPmLBCv4/edit?usp=sharing

File: e2e/mtav6.2.2.sh
Comment (rhkp) [reply e2e/mtav6.2.2.sh:1]: Things are wip, please see my latest comment.

File: mtav7.0.2/application.properties (hunk 4 above)
Comment (dmartinol) [reply mtav7.0.2/application.properties:4]: Then I give up, surely @gabriel-farache knows it

**Comment by rhkp** [comment]
Closing this for now as needs local tests and to avoid GH Actions pipeline triggers for every commit.

**Pull request closed without merging**